CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# v1.13.3
+
+## Release v1.13.3 - 2021/07/27
+
+### Bug fix
+
+* in_tail: Care DeletePending state on Windows
+  https://github.com/fluent/fluentd/pull/3457
+  https://github.com/fluent/fluentd/pull/3460
+* in_tail: Fix some pos_file bugs.
+  Avoid deleting pos_file entries unexpectedly on compaction with `follow_inode`.
+  Use `bytesize` instead of `size` for path length.
```

Comment:
Is there no need to mention `pos_file_compaction_interval` here ?

Reply:
Fixed.

Reply:
forgot to push fixed version?

Reply:
Oops, pushed again.